android/src/main/java/guichaguri/trackplayer/player/Playback.java

```diff
@@ -134,6 +134,8 @@ public void reset() {
 
     public abstract void play();
 
+    public abstract void playWithEarPiece();
```

Comment:
I think passing it as an option instead of a new function would be better

Reply:
I was thinking about this. I'm not sure if there will be more options in a future. But yeah, I will do it like you said. I also like the "parameter solution" ...